Library/OcCpuLib/OcCpuLib.c

```diff
@@ -956,8 +982,8 @@ OcCpuScanProcessor (
     "OCCPU: %a %a %11lld %5dMHz\n",
```

Comment:
There is an issue with formatting again.

Reply:
Fixed in latest.